lib/puppet/functions/find_template.rb

```diff
@@ -0,0 +1,63 @@
+# Finds an existing template from a module and returns its path.
+#
+# The argument to this function should be a String as a `<MODULE NAME>/<TEMPLATE>`
```

Comment:
Change "The argument to this function should be" to "This function accepts an argument that is"